### PR TITLE
Split big switch to methods calls, remove dulicate code

### PR DESCRIPTION
**Pull request by proggga**

### CODE REVIEW

**Comment by quasilyte** [comment]
@i582 could you do a review?

**Comment by i582** [comment]
> @i582 could you do a review?

yep

File: src/langsrv/defs.go [hunk 1]
```diff
@@ -44,225 +44,233 @@ func (d *definitionWalker) EnterNode(n ir.Node) bool {
...
+	var fun meta.FuncInfo
+	var ok bool
+	var nameStr string
 
-		c, ok := meta.Info.GetClassOrTrait(className)
+	if nm, isName := n.Function.(*ir.Name); isName {
+		nameStr = nm.Value
+		if nm.IsFullyQualified() {
+			fun, ok = meta.Info.GetFunction(nameStr)
+		} else {
```
Comment (i582) [inline src/langsrv/defs.go:220]: Please remove this blank line :)

Comment (proggga) [reply src/langsrv/defs.go]: Done

**Review by i582 — APPROVED** [review]

**Review by quasilyte — APPROVED** [review]

**Pull request merged**